### DISCUSSION TITLE
Chords adjusted via NextInVoice relations

### DISCUSSION THREAD

**Discussion by hbitteur** [Audiveris/audiveris#501] [General]
- Chords popup menu offers "Next In Voice" relation to better guide voice/slot assignments
- Improved tie slurs detection on staff, across systems and across pages
- Fixed regression on some head/stem configurations in 5.2.1
- Minor bug fixes 

<hr /><em>This discussion was created from the release <a href='https://github.com/Audiveris/audiveris/releases/tag/5.2.2'>Chords adjusted via NextInVoice relations</a>.</em>

**Comment by k-l-lambda** [comment]
Audiveris didn't launch after program startup click, the windows installation. Should I install JVM or something?

**Reply by hbitteur** [reply]
Yes, you need Java 11
Please read installation doc...

**Reply by k-l-lambda** [reply]
Thanks!
